src/jv.c

```diff
@@ -535,9 +535,13 @@ static decContext* tsd_dec_ctx_get(pthread_key_t *key) {
     }
     else if (key == &dec_ctx_dbl_key)
     {
-      decContextDefault(ctx, DEC_INIT_DECIMAL64);
-      // just to make sure we got this right
-      assert(ctx->digits <= DEC_NUBMER_DOUBLE_PRECISION);
+      // constants suitable for double (binary64)
+      ctx->digits = DEC_NUBMER_DOUBLE_PRECISION;
+      ctx->emax = 1023;
+      ctx->emin = -1022;
+      ctx->round = DEC_ROUND_HALF_EVEN;
+      ctx->traps = 0;
```

Comment:
As i understand the decnum code traps are used to SIGFPE on some error conditions, probably not what we want and not what we do for dec_ctx_key above.